src/incomingcallnotifications.cs

```diff
@@ -0,0 +1,114 @@
+//
+// IncomingCallNotifications C# bindings
+//
+// Authors:
+//	TJ Lambert  <t-anlamb@microsoft.com>
+//
+// Copyright 2019 Microsoft Corporation.
+//
+
+using System;
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+
+namespace IncomingCallNotifications {
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[Native]
+	enum ICHandleType : long {
+		Generic = 1,
+		PhoneNumber = 2,
+		EmailAddress = 3,
+	}
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[Native]
+	enum ICDeclineReason : long {
+		Generic,
+		Cancelled,
+		UserBusy,
+		Unanswered,
+	}
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface ICHandle : NSCopying {
+
+		[Export ("type")]
+		ICHandleType Type { get; }
+
+		[Export ("value", ArgumentSemantic.Copy)]
+		string Value { get; }
+
+		[DesignatedInitializer]
+		[Export ("initWithType:value:")]
+		IntPtr Constructor (ICHandleType type, string value);
+
+		[Export ("isEqualToHandle:")]
+		bool IsEqual (ICHandle handle);
+	}
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface ICNotification : NSCopying {
+
+		[DesignatedInitializer]
+		[Export ("initWithUUID:handle:")]
+		IntPtr Constructor (NSUuid identifier, ICHandle handle);
+
+		[Export ("identifier", ArgumentSemantic.Copy)]
+		NSUuid Identifier { get; }
+
+		[Export ("handle", ArgumentSemantic.Copy)]
+		ICHandle PersonHandle { get; }
+
+		[Export ("video", ArgumentSemantic.Assign)]
+		bool Video { [Bind ("isVideo")] get; set; }
+	}
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[BaseType (typeof (NSObject))]
+	[Model (AutoGeneratedName = true)] [Protocol]
+	interface ICNotificationManagerDelegate {
+
+		[Export ("notificationManager:didAcceptNotification:")]
+		void DidAcceptNotification (ICNotificationManager notificationManager, ICNotification notification);
```

Comment:
Notification is redundant. DidAccept, since you get that is form the notification manager and a notification.